lib/logstash/inputs/kafka.rb

```diff
@@ -50,6 +50,8 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   # have an established offset or offset is invalid, start with the earliest message present in the
   # log (`smallest`) or after the last message in the log (`largest`).
   config :auto_offset_reset, :validate => %w( largest smallest ), :default => 'largest'
+  # The frequency in ms that the consumer offsets are committed to zookeeper.
+  config :auto_commit_interval_ms, :validate => :number, :default => 60 * 1000
```

Comment:
this config is `auto.commit.interval.ms` in kafka, but it is `auto_commit_interval` for when we pass this into jruby-kafka